packages/patternfly-4/react-core/src/components/AlertGroup/AlertGroup.tsx

```diff
@@ -0,0 +1,53 @@
+import styles from '@patternfly/react-styles/css/components/AlertGroup/alert-group';
+import { css } from '@patternfly/react-styles';
+import { canUseDOM } from 'exenv';
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import { Omit } from '../../helpers/typeUtils';
+
+export interface AlertGroupProps extends Omit<React.HTMLProps<HTMLUListElement>, 'className'> {
+  /** Additional classes added to the AlertGroup */
+  className?: string;
+  /** Alerts to be rendered in the AlertGroup */
+  children?: React.ReactNode;
+  /** Toast notifications are positioned at the top right corner of the viewport */
+  isToast?: boolean;
+}
+
+class AlertGroup extends React.Component<AlertGroupProps> {
+  container?: HTMLDivElement = undefined;
+  constructor(props: AlertGroupProps) {
+    super(props);
+  }
+  componentDidMount() {
+    if (this.container) {
+      document.body.appendChild(this.container);
```

Comment:
I think we should add a prop so user can define where this should be appended to, like in AboutModal something like:`appendTo?: HTMLElement | (() => HTMLElement);`

Reply:
Sounds good to me! :)